essm/_generator.py

```diff
@@ -129,9 +129,9 @@ class VariableWriter(object):
     """Generate Variable definitions.
 
     Example:
-
```

Comment:
@schymans is it necessary to change the formatting here?

Reply:
Travis complained that there should be no blank line below a new section. Not sure if this classifies as a new section.

Reply:
OK, thanks for fixing it!